src/tribler-core/tribler_core/modules/metadata_store/orm_bindings/torrent_state.py

```diff
@@ -14,6 +14,7 @@ class TorrentState(db.Entity):
         seeders = orm.Optional(int, default=0)
         leechers = orm.Optional(int, default=0)
         last_check = orm.Optional(int, size=64, default=0)
+        self_checked = orm.Optional(bool, default=False)
```

Comment:
Currently, PonyORM applies defaults in Python and does not add `default` clauses to columns in the database automatically (because in PonyORM default may be arbitrary Python function).
It is better to add a second option `sql_default="0"` to this attribute. Otherwise, upgrade from 7.5 to 7.6 will put NULL instead of 0 to this column for previously existed records.
It is not critical for the current logic that processes this attribute, but this way the data will be more consistent.

Reply:
Good to know, thanks